app_builder/helpers.py

```diff
@@ -1,4 +1,11 @@
 import re
 
 def parse_version_string(version_string):
-    return re.split("[\.-]", version_string.strip())[:3]
\ No newline at end of file
+    return re.split("[\.-]", version_string.strip())[:3]
+
+def compare_versions(sublime_required_version, cli_version):
+    result = True
+    for index, item in enumerate(sublime_required_version):
+        if(item != cli_version[index] and item != "*"):
```

Comment:
A bit of nitpicking, but I believe this check will succeed for "2.*.3". Do we want to allow this?